.github/workflows/ready_for_review.yml

```diff
@@ -161,13 +160,17 @@ jobs:
   fetch-pr-reviews:
     name: Fetch Pull Request Reviews
     needs: [get-pr-data, check-workflow-status, check-backend-requirement]
-    if: needs.check-workflow-status.outputs.failures_detected == 'false' && needs.check-backend-requirement.outputs.backend_approval_required == 'true'
+    # if: needs.check-workflow-status.outputs.failures_detected == 'false'
```

Comment:
Can this be fully removed?

Reply:
I will be updating the if statements when I am finished testing